src/main/java/org/graylog/plugins/outputs/neo4j/Neo4jOutput.java

```diff
@@ -151,9 +104,18 @@ public void stop() {
         public ConfigurationRequest getRequestedConfiguration() {
             final ConfigurationRequest configurationRequest = new ConfigurationRequest();
 
+            final Map<String, String> protocols = ImmutableMap.of(
+                    "HTTP", "HTTP",
+                    "Bolt", "Bolt");
+
+            configurationRequest.addField(new DropdownField(
+                            CK_PROTOCOL, "Neo4J Protocol", "HTTP", protocols,
+                            "The protocol used to connect to Neo4J",
+                            ConfigurationField.Optional.NOT_OPTIONAL));
+
             configurationRequest.addField(new TextField(
                             CK_NEO4J_URL, "Neo4j URL", "http://localhost:7474/db/data",
-                            "URL to Neo4j",
+                            "URL to Neo4j, default for Bolt: bolt://localhost:7687/",
```

Comment:
Try to keep the field label as short as possible and use the field footer for examples.